community/github/continuous-integration.md

```diff
@@ -34,6 +34,9 @@ checks:
   can be merged. The issues could be related to code style, formatting, tests,
   or other criteria specified by the CI configuration.
 
+![When a CI check fails, you'll see a red 'X' next to the CI check that hasn't passed](/images/ci-images/contributor-ci-fail.png)
```

Comment:
@kierisi it's important that all of our documented are accessible. As such please add both an alt tag for accessibility and a caption. Here i think this is a caption, but it actually reads as an alt tag in the html rendered page. 

You can either look at the myst markdown documentation for how to do this or look at how we add images in the packaging guide for guidance.